[2024-02-19T17:21:17.114+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: primeira_dag.tsk1 manual__2024-02-19T17:21:16.425005+00:00 [queued]>
[2024-02-19T17:21:17.123+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: primeira_dag.tsk1 manual__2024-02-19T17:21:16.425005+00:00 [queued]>
[2024-02-19T17:21:17.123+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-02-19T17:21:17.123+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-02-19T17:21:17.123+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-02-19T17:21:17.134+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk1> on 2024-02-19 17:21:16.425005+00:00
[2024-02-19T17:21:17.140+0000] {standard_task_runner.py:55} INFO - Started process 488 to run task
[2024-02-19T17:21:17.145+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'primeira_dag', 'tsk1', 'manual__2024-02-19T17:21:16.425005+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/primeiradag.py', '--cfg-path', '/tmp/tmpghljcx80']
[2024-02-19T17:21:17.146+0000] {standard_task_runner.py:83} INFO - Job 16: Subtask tsk1
[2024-02-19T17:21:17.207+0000] {task_command.py:388} INFO - Running <TaskInstance: primeira_dag.tsk1 manual__2024-02-19T17:21:16.425005+00:00 [running]> on host 43942cc1ca68
[2024-02-19T17:21:17.275+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=primeira_dag
AIRFLOW_CTX_TASK_ID=tsk1
AIRFLOW_CTX_EXECUTION_DATE=2024-02-19T17:21:16.425005+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-19T17:21:16.425005+00:00
[2024-02-19T17:21:17.276+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-02-19T17:21:17.277+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep']
[2024-02-19T17:21:17.285+0000] {subprocess.py:86} INFO - Output:
[2024-02-19T17:21:17.288+0000] {subprocess.py:93} INFO - sleep: missing operand
[2024-02-19T17:21:17.288+0000] {subprocess.py:93} INFO - Try 'sleep --help' for more information.
[2024-02-19T17:21:17.288+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-02-19T17:21:17.297+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-02-19T17:21:17.299+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=primeira_dag, task_id=tsk1, execution_date=20240219T172116, start_date=20240219T172117, end_date=20240219T172117
[2024-02-19T17:21:17.308+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 16 for task tsk1 (Bash command failed. The command returned a non-zero exit code 1.; 488)
[2024-02-19T17:21:17.315+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-02-19T17:21:17.330+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
